New validations and fixes for the oracle script

sfc-gh-aramirezfuentes opened the pull request:
- Updated the Oracle Readme.md to include the new parameter  OUTPUT_PATH
- Added new empty validation path for the scripts.
- Added a new parameter in the sql*plus version to support output path for the generated files.
- Adjusted the LONGCHUNCKSIZE parameter for the sql script to avoid tab empty spaces.

sfc-gh-aramirezfuentes committed:
- Updated the Oracle Readme.md to include the new parameter  OUTPUT_PATH.
- Added new empty validation path for the scripts.
- Added a new parameter in the sql*plus version to support output path for the generated files.
- Adjust the LONGCHUNCKSIZE parameter for the sql script to avoid tab empty spaces.

sfc-gh-mrojas commented on Oracle/README.md:
```diff
@@ -31,11 +31,11 @@ The following are the steps to execute the DDL Code Generation. They can be exec
+SQL\*Plus: If you have Oracle installed on your machine, you can use SQL\*Plus, which is a command-line tool for Oracle Database. It provides a powerful and interactive interface to execute SQL statements and scripts. 
 
-sqlcl: If you don't have Oracle installed locally or prefer a modern command-line interface, you can use sqlcl (SQL Developer Command Line). Sqlcl is a free, lightweight tool provided by Oracle that offers a feature-rich SQL and PL/SQL scripting environment. It provides similar functionality to Sql*Plus but with additional enhancements. 
+sqlcl: If you don't have Oracle installed locally or prefer a modern command-line interface, you can use sqlcl (SQL Developer Command Line). Sqlcl is a free, lightweight tool provided by Oracle that offers a feature-rich SQL and PL/SQL scripting environment. It provides similar functionality to SQL\*Plus but with additional enhancements. 
```
Can you add links to both of these tools, specially so clients can check any requirements needed

sfc-gh-mrojas commented on Oracle/README.md:
```diff
@@ -47,6 +47,7 @@ sqlcl: If you don't have Oracle installed locally or prefer a modern command-lin
+* `OUTPUT_PATH`
   
 4. After modifying, the `create_ddls.sh` file can be run from the command line to execute the extract. 
    In order to run the script you can open a terminal in the  `create_ddls.sh` path  and execute the command:
```
folder where  `create_ddls.sh` file is located

sfc-gh-mrojas commented on Oracle/README.md:
```diff
@@ -91,8 +93,12 @@ Optionally modify these parameters (see comments in the file for explanation of
 
+* The `OUTPUT_PATH` is the directory where the script results will be generated. This directory contains the files generated with the script results. By default, the `OUTPUT_PATH` variable is set to `SCRIPT_PATH`.
+
 * The `SQLCL_PATH` is the path where the `sql` file is located within the folder that you downloaded and extracted previously from the sqlcl download link. Usually the path is `<extracted_files>/bin/`.
```
This process is tricky we need to provide better guidance

sfc-gh-aramirezfuentes replied:
Some extra information added.

sfc-gh-aramirezfuentes committed:
- Try to generate the output path and verify if exist.
- Added some links to sqlcl and sqlplus documentation for reference.

sfc-gh-aramirezfuentes committed:
- added @ in the script path.
- removed unnecessary " in the validations.
- Updated Readme.md with restriction for OUTPUT_PATH

sfc-gh-aramirezfuentes committed:
- Added new section to the readme.md for Usage.
- Added version tested for the scripts.
- Added some default values for variables

sfc-gh-ogarciabarquero commented on Oracle/README.md:
```diff
@@ -1,41 +1,61 @@
 
-* If you want to use Sql*Plus, you need to have installed Oracle in you PC. 
-  To verify that SQL*Plus is installed and available for use, you can run a command to check its installation status. Open a command prompt or terminal and enter the following command:
+* If you want to use SQL\*Plus, you need to have installed SQL\*Plus in you PC. 
```
in your PC

sfc-gh-aramirezfuentes committed:
Fixed a spelling mistake in the readme.md

sfc-gh-dperezarroyo commented on Oracle/README.md:
```diff
@@ -171,29 +203,49 @@ These files will contain the definitions of the objects specified by the file na
+
+To address this issue, locate the LONGCHUNKSIZE variable declaration in the respective file (`create_ddls.sql` for SQLcl or `create_ddls_plus.sql` for SQL\*Plus). Modify the value of LONGCHUNKSIZE to a larger value that suits your requirements. Save the file and rerun your SQL utility.
+
+By increasing the LONGCHUNKSIZE, you provide more memory for handling long data, which can help resolve problems related to unnecessary tab spaces or truncated code. Remember to consider the specific needs and limitations of your environment when adjusting this variable.
```
I am just a bit curious about what it means "your specific needs", how can the user know what a correct value for this variable should be after getting these unnecessary blank spaces? What would happen if the user increases the variable to a gigantic value? Would that solve the issue all the time or that would introduce another problem?

sfc-gh-dperezarroyo replied:
Maybe we could guide a bit the user to a possible correct value or at least explain what happens if the value is too large

sfc-gh-aramirezfuentes replied:
there isn't a specify value or usual value. All depends of your PC settings, like ram, PC processor, internet speed ... right now It sets now  with a "Big number" because the default is 80. so, there isn't a recommended value or common value, depends of your computer.

sfc-gh-aramirezfuentes replied:
I added a little tip to increase and test the value.

sfc-gh-aramirezfuentes committed:
- Added new anotation for the longchunksize parameter.
- Added description in the README.md for the downloaded script zip.
- Added more information about the context in the README.md
